assets/js/components/ReportErrorActions.js

```diff
@@ -173,6 +196,12 @@ ReportErrorActions.propTypes = {
 		PropTypes.object,
 	] ).isRequired,
 	GetHelpLink: PropTypes.elementType,
+	showGetHelpLink: PropTypes.bool,
+	buttonVariant: PropTypes.string,
 	onRetry: PropTypes.func,
 	getHelpClassName: PropTypes.string,
 };
+
+ReportErrorActions.defaultProps = {
+	showGetHelpLink: true,
+};
```

Comment:
`defaultProps` is deprecated in the latest React version, though we use it in our codebase. You can either pass a default value to the `showGetHelpLink` prop or rename it to `hideGetHelpLink` and use it accordingly, i.e., render the link if it's falsy.